LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageDispatcher.cs

```diff
@@ -44,11 +44,15 @@ public void DispatchRequest(LoRaRequest request)
         {
             if (request is null) throw new ArgumentNullException(nameof(request));
 
-            if (!LoRaPayload.TryCreateLoRaPayload(request.Rxpk, out var loRaPayload))
+            if (request.Payload is null)
```

Comment:
nit: maybe we could add a comment that this is a legacy path to not forget to clean it up

Reply:
Comment added